packages/mermaid/src/accessibility.ts

```diff
@@ -45,26 +44,20 @@ export function setA11yDiagramInfo(svg: D3Element, diagramType: string | null |
  */
 export function addSVGa11yTitleDescription(
   svg: D3Element,
-  a11yTitle: string | null | undefined,
-  a11yDesc: string | null | undefined,
+  a11yTitle: string | undefined,
+  a11yDesc: string | undefined,
   baseId: string
-) {
-  if (svg.insert === undefined) {
-    return;
-  }
-
-  if (a11yTitle || a11yDesc) {
+): void {
+  if (svg.insert !== undefined) {
     if (a11yDesc) {
```

Comment:
Return early is better than an extra nesting inside if. 

```ts
function test(param){
   if(checkCondition(param)){
      // rest of code
   }
}

function betterTest(param){
   if(!checkCondition(param)){
      return;
   }
   // rest of code is one intent level lesser. Lesser cognitive load in bigger functions.
}

Reply:
I only meant the initial condition, like this.

<img width="768" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/6c835e5e-8ae8-4dd8-9a6a-51e64e4fc85d">

Reply:
Fixed.